src/redist/targets/BundledManifests.targets

```
@@ -6,7 +6,7 @@
     <BundledManifests Include="Microsoft.NET.Sdk.MacCatalyst.Manifest-6.0.100" Version="$(XamarinMacCatalystWorkloadManifestVersion)" WorkloadManifestId="Microsoft.NET.Sdk.MacCatalyst" />
     <BundledManifests Include="Microsoft.NET.Sdk.macOS.Manifest-6.0.100" Version="$(XamarinMacOSWorkloadManifestVersion)" WorkloadManifestId="Microsoft.NET.Sdk.macOS" />
     <BundledManifests Include="Microsoft.NET.Sdk.tvOS.Manifest-6.0.100" Version="$(XamarinTvOSWorkloadManifestVersion)" WorkloadManifestId="Microsoft.NET.Sdk.tvOS" />
-    <BundledManifests Include="Microsoft.NET.Sdk.BlazorWebAssembly.AOT" Version="$(BlazorWorkloadManifestVersion)" WorkloadManifestId="Microsoft.NET.Workload.BlazorWebAssembly" />
+    <BundledManifests Include="Microsoft.NET.Workload.Mono.ToolChain.Manifest-6.0.100" Version="$(MonoWorkloadManifestVersion)" WorkloadManifestId="Microsoft.NET.Workload.Mono.ToolChain.Manifest" />
```

I think the path on disk should be:
```suggestion
    <BundledManifests Include="Microsoft.NET.Workload.Mono.ToolChain.Manifest-6.0.100" Version="$(MonoWorkloadManifestVersion)" WorkloadManifestId="Microsoft.NET.Workload.Mono.ToolChain" />
```

---

@dsplaisted What is the right value here?

---

For example, we want to double check if the package: `Microsoft.NET.Workload.Android.Manifest-6.0.100`

Should be at this location on disk: `dotnet\sdk-manifests\6.0.100\Microsoft.NET.Sdk.Android`

---

So the extra Manifest on the id is wrong and we can change back to the Sdk naming for everything but let'd get full agreement and make sure the documentation agrees before we do.

---

The manifest ID is `Microsoft.NET.Workload.Android` (I think, it's up to you to choose that ID).

Based on that ID, the path on disk would be in `dotnet\sdk-manifests\6.0.100\Microsoft.NET.Workload.Android` (ie using the manifest ID as a folder)

The NuGet package that includes the manifest also gets `.Manifest-{version-band}` appended to it, so the NuGet package ID would be `Microsoft.NET.Workload.Android.Manifest-6.0.100`.

See the "Packaging" section in this PR: https://github.com/dotnet/designs/pull/188/files

---

I don't know how this works exactly, but if the folder name is wrong would `dotnet workload install` end up with two duplicate directories? Duplicate `WorkloadManifest.json` files would cause `dotnet build` to fail.

I don't think the actual package needs to change, though.

---

So I think this PR just needs to drop `.Manifest` off the end:
```suggestion
    <BundledManifests Include="Microsoft.NET.Workload.Mono.ToolChain.Manifest-6.0.100" Version="$(MonoWorkloadManifestVersion)" WorkloadManifestId="Microsoft.NET.Workload.Mono.ToolChain" />
```
And then I should adjust the names of the mobile workloads so they are `Microsoft.NET.Workload.Android`, for example.

Does that sound right?

---

Makes sense to me.

---

Sounds good to me too.